internal/service/encryptionatrest/resource_encryption_at_rest_migration_test.go

```diff
@@ -57,20 +60,19 @@ func TestMigEncryptionAtRest_basicAWS(t *testing.T) {
 }
 
 func TestMigEncryptionAtRest_withRole_basicAWS(t *testing.T) {
-	acc.SkipTestForCI(t) // needs AWS configuration
+	acc.SkipTestForCI(t) // TODO: enable in CI as part of CLOUDP-267663
```

Comment:
updated the config for this so this test can be enabled as part of  CLOUDP-267663 itself without much extra effort